src/gross_to_net_generation.py

```diff
@@ -367,16 +379,33 @@ def calculate_gross_to_net_conversion_factors(
         gtn_regression_plant, how="left", on=["plant_id_eia"], validate="m:1"
     )
 
+    # add default GTN ratios from EIA
+    gtn_conversions = gtn_conversions.merge(
+        load_data.load_default_gtn_ratios(),
+        how="left",
+        on="prime_mover_code",
+        validate="m:1",
+    )
+    # for any misisng default gtn ratios, use a ratio of 0.97
+    gtn_conversions["default_gtn_ratio"] = gtn_conversions["default_gtn_ratio"].fillna(
```

Comment:
This default-filling logic occurs twice (here and in convert_gross_to_net_generation). Should only be in one place. Also, as commented above, maybe we should throw a warning instead of giving a default value?

Reply:
Good catch. I think that we want to throw a warning but also probably use the default fill number for now, just so that we don't get any NA values when doing the multiplication. But I agree we should only be doing this filling in a single spot.